charts/seed-bootstrap/charts/monitoring/templates/config.yaml

```diff
@@ -143,3 +148,36 @@ data:
           target_label: pod
         - source_labels: [ __meta_kubernetes_namespace ]
           target_label: namespace
+
+    - job_name: gardenlet
+      scheme: https
```

Comment:
You could switch to `http` directly here instead of overwriting it later on via `prometheus.io/scheme`.
I.e., drop the `scheme` setting here (which means `http`) and then drop usage of the `prometheus.io/scheme` annotation below.

Reply:
Addressed in https://github.com/gardener/gardener/pull/7180/commits/41d8d6631f039ef48cc0c4f8d32562e9b0c2a23f